sky/serve/serve_utils.py

```diff
@@ -262,6 +269,20 @@ def update_service_status() -> None:
                 record['name'], serve_state.ServiceStatus.CONTROLLER_FAILED)
 
 
+def update_service(controller_port: int, version: int,
+                   mixed_replica_versions: bool) -> str:
+    resp = requests.post(
+        _CONTROLLER_URL.format(CONTROLLER_PORT=controller_port) +
+        '/controller/update_service',
+        json={
+            'version': version,
+            'mixed_replica_versions': mixed_replica_versions
+        })
+    if resp.status_code != 200:
+        raise ValueError(f'Failed to update service: {resp.text}')
```

Comment:
We should handle a special case here: if the status code is 404, i.e. there is no such endpoint in the HTTP server, which indicates it is a service `up`-ed before this PR, print an hint message saying old services does not support update and pls `sky serve down` first and re-launch.